fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -1098,6 +1102,32 @@ protected DruidWebService buildMetadataDruidWebService(ObjectMapper mapper) {
         return buildDruidWebService(DruidClientConfigHelper.getMetadataServiceConfig(), mapper);
     }
 
+    /**
+     * Create a ResponseFormatResolver for Servlet objects.
+     * <p>
+     * Currently default types are json, jsonapi and csv types.
+     *
+     * @return A ResponseFormatResolver
+     */
+    protected ResponseFormatResolver buildResponseFormatResolver() {
+        return (format, containerRequestContext) -> {
+            Map<String, String> formatsMap = new LinkedHashMap<>();
```

Comment:
Build the map above the return so it's not being rebuilt every time, then close over the map.